packages/test/test-gc-sweep-tests/src/dataObjectWithCounter.ts

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IRandom } from "@fluid-internal/stochastic-test-utils";
+import { DataObject } from "@fluidframework/aqueduct";
+import { assert } from "@fluidframework/common-utils";
+import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { SharedCounter } from "@fluidframework/counter";
+
+/**
+ * DataObjectWithCounter increments a SharedCounter as a way of sending ops.
+ *
+ * The SharedCounter is retrieved via handle
+ */
+const counterKey = "counter";
+export class DataObjectWithCounter extends DataObject {
+    private _counterHandle?: IFluidHandle<SharedCounter>;
+    public isRunning: boolean = false;
```

Comment:
Why is this needed?

Reply:
Good point, I added it to the sendOp to prevent ops from being sent when the dataobject is stopped.